### ISSUE TITLE
[Request]: Individually Timeout Players

### ISSUE THREAD

**Issue by Rxann** [Yimura/YimMenu#1844]
### Problem

No, it's just that you can only timeout people right now via reactions and I would like to see that implemented for every player.

### Solution

A checkbox or something to enable timing them out. maybe in the toxic or kick menu idk.

### Reason

It is available as a reaction, so it should probably be available on an individual player basis.

**Comment by sch-lda** [comment]
![Screenshot_2023-07-26-09-30-52-71_984e1414ae90666a90f12ff17ec14a7f](https://github.com/YimMenu/YimMenu/assets/54973190/53e222f6-94f0-46b8-8335-ce88f03e28af)
You mean block all players‘  sync？This might be a solution

**Comment by sch-lda** [comment]
You can select one player in Yimmenu player list , then click More Info, select block net events,block clone sync manually.

**Comment by Rxann** [comment]
yes, but just something more together would be nice

**Comment by Yimura** [comment]
Could you further elaborate on why this should be available as a normal checkbox instead of only having it as a reaction?

**Comment by Rxann** [comment]
Yes, It should be a normal checkbox because if someone does something that I don't like, I can time them out instead of having to wait for something they do to be handled by a reaction.

**Comment by Tayipc38** [comment]
I agree with this request.

**Comment by spinoza-spinoza** [comment]
hi 
please what timeout  mean and  do  if i check that case

**Comment by sch-lda** [comment]
> hi please what timeout mean and do if i check that case

Block others players' network events, clone sync. Prevent most of the cheaters' attacks without the need for kicking them out.

**Comment by spinoza-spinoza** [comment]
ok tank you   should   i check all   timeout  case    in reaction ?? for example   report or spectate or destroy my car

**Comment by Yimura** [comment]
Well report can be a legit player and not a modder, so giving those people a timeout would be a stupid thing to do, I would honestly only timeout someone for extreme things like crashes.

**Comment by spinoza-spinoza** [comment]
ok .tank you for reply 
i will do it just for kick .crash or control my car or  teleporting me 
i  had 2 week  use this menu  and none reported me or vote cickin me
i use it  just for 2 modder who still following me for 2 month to crash me in publique

**Comment by spinoza-spinoza** [comment]
hi 
do you have any documentation or help  for   yimmenu
i need  it to understand  some issu
honnestly i dont know what mean  all reactions    and when i  use it  
for example   sent to enterior  .nse freze .....remote ragdol
